app/models/remote_execution_provider.rb

```diff
@@ -35,6 +35,10 @@ def humanized_name
       self.name
     end
 
+    def provider_input_namespace
+      []
```

Comment:
From the name of the method I'd expect a single value, such as a symbol or a string. Why is it an array? Are elements of the array components of the namespace or can a provider have multiple input namespaces?

Reply:
Maybe I am making it more complicated than it needs to be, changed.